pages/common/ssh.md

```diff
@@ -25,8 +25,12 @@
 
 - SSH tunneling: Forward a specific port (localhost:9999 to slashdot.org:80):
 
-`ssh -L {{9999}}:slashdot.org:80 {{username}}@{{remote_host}}`
+`ssh -L {{9999}}:slashdot.org:80 -N -T {{username}}@{{remote_host}}`
```

Comment:
Sorry, one last thing. Could you explain the `-N` and `-T` options in the description ? We usually describe all the options used in the description so that user has a clear idea what is going on.